pkg/api/validation/schema.go

```diff
@@ -66,6 +67,48 @@ func NewSwaggerSchemaFromBytes(data []byte) (Schema, error) {
 	return schema, nil
 }
 
+func (s *SwaggerSchema) validateList(obj map[string]interface{}) error {
+	items, exists := obj["items"]
+	if !exists {
+		return fmt.Errorf("no items field in %#v", obj)
+	}
+	if _, ok := items.([]interface{}); !ok {
+		return fmt.Errorf("items isn't a slice")
+	}
+	for _, item := range items.([]interface{}) {
```

Comment:
Yes, I'm thinking about this too. It's better to check every `item` in one time.